File: leetcode/0151-reverse-words-in-a-string/solution.cpp
class Solution {
public:
    string reverseWords(string s) {
        string result;

        for (int i=0;i<s.size();i++){
            if(s[i]==' '){
                continue;
            }
            string word;
            while(i<s.size() && s[i]!=' '){
                word+=s[i];
                i++;
            }
            if(result.empty()){
                result=word;
            }
            else{
            result=word+' '+result;
            }
        }
        return result;
    }
};

File: leetcode/0151-reverse-words-in-a-string/solution.py
class Solution(object):
    def reverseWords(self, s):
        reverse=""
        i=0
        while i<len(s):
            if(s[i]==' '):
                i=i+1
                continue
            else:
                word=""
                while(i<len(s) and s[i]!=" "):
                    word=word+s[i]
                    i=i+1
            if(reverse==""):
                reverse=word
            else:
                reverse=word+" "+reverse
            i=i+1
        return reverse
        """
        :type s: str
        :rtype: str
        """
        
